README.md

```diff
@@ -77,8 +79,11 @@ pip_parse(
 )
 ```
 
-> You may find some quirks while using this toolchain.
-> Please refer to [this link](https://python-build-standalone.readthedocs.io/en/latest/quirks.html) for details.
```

Comment:
Within my company's internal writing style guide, we call this "descriptive link text". 

> Avoid terms like “here” or “this link” in link text.